router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -388,33 +428,24 @@ func getFailedReasons(clientIDErrors map[string]map[string]struct{}) map[string]
 	return reasons
 }
 
-// filtering out failed jobIds from the total array of jobIds
-// in order to get jobIds of the successful jobs
-func getSuccessKeys(failedEventList, initialEventList []int64) []int64 {
-	successfulEvents := make([]int64, 0)
-
-	lookup := make(map[int64]bool)
-	for _, element := range failedEventList {
-		lookup[element] = true
-	}
-
-	for _, element := range initialEventList {
-		if !lookup[element] {
-			successfulEvents = append(successfulEvents, element)
-		}
-	}
-
-	return successfulEvents
-}
-
-func (b *BingAdsBulkUploader) FetchFailedEvents(failedJobsStatus common.FetchFailedStatus) ([]byte, int) {
-	filePath := failedJobsStatus.OutputFilePath
+var ReadPollResults = func(filePath string) [][]string {
```

Comment:
func ReadPollResults(filePath string) [][]string { is better right?

Reply:
addressed